indico/modules/users/templates/users/dashboard.html

```diff
@@ -282,7 +248,12 @@
                         method: 'PUT',
                         error: handleAjaxError,
                         success: function() {
-                            container.find('.actions, .close-box').remove();
+                            container.find('.category-action-extra').remove();
+                            container.find('.category-action').replaceWith([
```

Comment:
$('<span>', {'class': 'category-action icon-star active', 'title': $T('You have favorited this category')})